src/SldStyleParser.ts

```diff
@@ -495,8 +509,10 @@ class SldStyleParser implements StyleParser {
             const filter: Filter | undefined = this.getFilterFromRule(sldRule);
             const scaleDenominator: ScaleDenominator | undefined = this.getScaleDenominatorFromRule(sldRule);
             const symbolizer: Symbolizer = this.getSymbolizerFromRule(sldRule);
-            let rule: Rule = <Rule> {};
-            rule = {
+            const name = sldRule.Title ? sldRule.Title[0]
+              : (sldRule.Name ? sldRule.Name[0] : '');
```

Comment:
Yes it would be cool if could have title and name separated and dont mix them up.
Example: when using a GetLegendGraphic request (against GeoServer), one can specify a specific rule to render.
You have to use the rule name there, the title will not work.

Reply:
Or we take ``Name`` over ``Title`` since ``Title`` is just a nice to have human readable thing?

Reply:
I choose `Title` over `Name` because of the [SLD documentation of the Geoserver](http://docs.geoserver.org/latest/en/user/styling/sld/reference/rules.html#syntax). The `Title` is used for legends so I thought this would be more usefull. 

Isn't it an option to take the same value for title and name?

I'm against adding a 'Title' property to the rule as we didn't want to copy the SLD spec.

Reply:
I am absolutely +1 with taking the same option for both tags. Keeps it simple and as stated we don't wanna copy the SLD spec.

Just for the record: I think the expression in the SLD documentation you linked ``The title is used in display lists and legends.`` is meant that the ``Title`` is shown within the legend itself. But the ``Name`` is used to referenced it (as far as I can remember).

Reply:
@chrismayer right, that's how it works. 
So again i would vote for support of both properties and not mixing them up, because they effectively do separate things

Reply:
As this seems to be not blocking but very important to you i opened an issue: 
https://github.com/terrestris/geostyler/issues/113

@hwbllmnn The i18n stuff could also go into an issue. Pleae create on here if wanted: https://github.com/terrestris/geostyler/issues and choose GeoStyler-SLD-Parser under projects.